system-variables.md

```diff
@@ -452,7 +452,17 @@ SET  GLOBAL tidb_distsql_scan_concurrency = 10;
 
 - 作用域：INSTANCE
 - 默认值：0
-- 这个变量用来设置是否在日志里记录所有的 SQL 语句。
+- 这个变量用来设置是否在[日志](tidb-configuration-file.md#logfile)里记录所有的 SQL 语句, 该功能默认关闭。该功能常在系统运维人员定位问题过程中需要追踪所有 SQL 记录的情况下被打开。
+- 通过查询 "GENERAL_LOG" 字符串可以定位到该功能记录的所有日志。日志会记录以下内容：
+	- conn: 当前会话对应的 ID
+	- user: 当前会话用户
+	- schemaVersion: 当前 schema 版本
+	- txnStartTS: 当前事务的开始时间戳
+	- forUpdateTS: 事务模型为悲观事务时，SQL 语句的当前时间戳。悲观事务内发生写冲突时，会重试当前执行语句，该时间戳会被更新。重试次数由 [max-retry-count](/tidb-configuration-file#max-retry-count) 配置。事务模型为乐观事务时，该条目与 txnStartTS 等价。 
+	- isReadConsistency: 当前事务隔离级别是否至少确保读已提交
```

Comment:
```suggestion
	- isReadConsistency: 当前事务隔离级别是否是读已提交（RC）
```

Reply:
Accepted in https://github.com/pingcap/docs-cn/pull/4843/commits/c00d3ca6df8cdfbc4ed31b5c5b6de34a1e0c5cfc.